src/data/registry.xml

```diff
@@ -1534,5 +1642,140 @@
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension vertical_layer_dimension</dimensions>
+      <initial_value>0._kind_phys</initial_value>
+      <ic_file_input_names>ICIWP pbuf_ICIWP</ic_file_input_names>
+    </variable>
+    <variable local_name="icswpth"
+              standard_name="in_cloud_snow_water_path"
+              units="kg m-2" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension vertical_layer_dimension</dimensions>
+      <initial_value>0._kind_phys</initial_value>
+      <ic_file_input_names>ICSWP pbuf_ICSWP</ic_file_input_names>
+    </variable>
+    <variable local_name="dei"
+              standard_name="effective_diameter_of_stratiform_cloud_ice_particle_for_radiation"
+              units="um" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension vertical_layer_dimension</dimensions>
+      <initial_value>0._kind_phys</initial_value>
+      <ic_file_input_names>DEI pbuf_DEI</ic_file_input_names>
+    </variable>
+    <variable local_name="des"
+              standard_name="effective_diameter_of_stratiform_snow_particle_for_radiation"
+              units="um" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension vertical_layer_dimension</dimensions>
+      <initial_value>0._kind_phys</initial_value>
+      <ic_file_input_names>DES pbuf_DES</ic_file_input_names>
+    </variable>
+    <variable local_name="qrs"
+              standard_name="tendency_of_dry_air_enthalpy_at_constant_pressure_due_to_shortwave_radiation_adjusted_by_air_pressure_thickness"
+              units="J Pa kg-1 s-1" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension vertical_layer_dimension</dimensions>
+      <initial_value>0._kind_phys</initial_value>
+      <ic_file_input_names>QRS pbuf_QRS</ic_file_input_names>
+    </variable>
+    <variable local_name="qrl"
+              standard_name="tendency_of_dry_air_enthalpy_at_constant_pressure_due_to_longwave_radiation_adjusted_by_air_pressure_thickness"
+              units="J Pa kg-1 s-1" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension vertical_layer_dimension</dimensions>
+      <initial_value>0._kind_phys</initial_value>
+      <ic_file_input_names>QRL pbuf_QRL</ic_file_input_names>
+    </variable>
+    <variable local_name="two"
+              standard_name="ccpp_constant_two"
+              units="count" type="integer"
+	      allocatable="parameter">
+      <initial_value>2</initial_value>
+    </variable>
+    <variable local_name="snow_exists"
+	      standard_name="include_liquid_plus_snow_stratiform_cloud_area_fraction"
+	    units="flag" type="logical">
+      <initial_value>.false.</initial_value>
+      <initial_value dyn="NONE"> .true. </initial_value>
```

Comment:
I actually might just default this to `.true.`, as most CAM-SIMA configurations are likely to have snow:


```suggestion
      <initial_value>.true.</initial_value>
```